sdk/src/timing.rs

```diff
@@ -80,6 +80,18 @@ impl AtomicInterval {
                 == Ok(last)
             && !(skip_first && last == 0)
     }
+
+    /// return ms elapsed since the last time the time was set
+    pub fn elapsed_ms(&self) -> u64 {
+        let now = timestamp();
+        let last = self.last_update.load(Ordering::Relaxed);
+        now.saturating_sub(last) // wrapping somehow?
+    }
```

Comment:
Can `should_update_ext()` be implemented using `elapsed_ms()` for its first 3ish lines?

Reply:
Nevermind!